### DISCUSSION TITLE
Simulating the ground effect

### DISCUSSION THREAD

**Discussion by ss-hegde** [byuflowlab/FLOWUnsteady#138] [General]
Hello all,

I am trying to simulate the propeller wake-ground interactions. I could find a function which generates the ground surface. 
```
"""
    Generates ground surface.
"""
function generate_ground_vahana(bscale; verify_spline=false, spl_s=0.001,
                                spl_k="automatic", visualize=false)
```
How should I define this ground in the simulation, so that the wake interacts with the ground surface?

Thanks in advance!

**Comment by weijunsjtu** [comment]
Hello, have you solved this problem yet. How did you define this ground function. Very much looking forward to your answer

**Comment by EdoAlvarezR** [comment]
The capabilities for modeling the ground with a panel method are available in the [`flowpanel-prototype` branch](https://github.com/byuflowlab/FLOWUnsteady/tree/flowpanel-prototype) but no documentation or examples are yet provided for users since these features are still under heavy development.

**Reply by Liuhaosheng1** [reply]
Hello author, is there currently a method of modeling the ground that can be provided? Or provide code for ground modeling?
